src/pilot/templates/DellNfv/numa.yaml

```diff
@@ -0,0 +1,29 @@
+heat_template_version: 2016-04-08
```

Comment:
Should add the standard licensing header to the top of this file.

Reply:
Added.